allensdk/brain_observatory/gaze_mapping/_gaze_mapper.py

```diff
@@ -297,31 +297,53 @@ def pupil_position_on_monitor_in_degrees(self,
 
 
 def compute_circular_areas(ellipse_params: pd.DataFrame) -> pd.Series:
```

Comment:
Why not compute a circular area with a more general `compute_elliptical_areas` function? This way only a single function is needed.

Reply:
`compute_circular_areas()` get ellipse fit `width` and `height` params. It determines for each frame in the video that the largest between `width` and `height` will be treated as the radius of a circle.

The above is not as straightfoward as the ellipse formula (np.pi * `width` * `height`) so I hesitate to combine the two.

Reply:
ok